pkgs/tools/audio/beets/default.nix

```diff
@@ -158,16 +167,7 @@ in pythonPackages.buildPythonApplication rec {
   ];
 
   patches = [
-    ./replaygain-default-bs1770gain.patch
```

Comment:
What's the reason for removing this patch?

Reply:
No reason. My bad. For some reason I thought it was like the "Fix Python 3.7 compatibility" patch that was now included upstream.